Themes Showcase: Replace empty search card with more theme options

mmtr opened the pull request:
Fixes https://github.com/Automattic/dotcom-forge/issues/1453

#### Proposed Changes

Replaces the previous upsell nudge displayed on empty searches, with a new card that offer users to perform the following actions:
- Design your own theme / homepage: Allow users to change their own homepage in the site editor (only in the logged-in showcase, and only if the site has a block theme active).
- Do it for me: Upsells the DIFM service.
- Upload a theme: Upsells the Business plan and/or nudges users to upload a third-party/custom theme missing.

**Screenshots**
<table>
<thead><tr><th>Conditions</th><th>Before</th><th>After</th></tr></thead>
<tbody>
<tr>
<td>Showcase: Logged-in<br><br>Plan: Free<br><br>Current theme: Block theme</td>
<td>
<img width="1001" alt="Screenshot 2023-02-02 at 14 49 25" src="https://user-images.githubusercontent.com/1233880/216342469-a324fc56-c6b2-4389-b608-a2987d88fa0f.png">
</td>
<td>
<img width="1006" alt="Screenshot 2023-02-03 at 11 15 23" src="https://user-images.githubusercontent.com/1233880/216574321-4d11c7b9-ce6c-4b95-980e-07f9e36aa4e5.png">
</td>
</tr>
<tr>
<td>Showcase: Logged-in<br><br>Plan: Business<br><br>Current theme: Block theme</td>
<td>
<img width="1005" alt="Screenshot 2023-02-02 at 14 52 14" src="https://user-images.githubusercontent.com/1233880/216343142-7ce3f302-0b86-4371-be7a-e30db5e51b93.png">
</td>
<td>
<img width="1006" alt="Screenshot 2023-02-03 at 11 16 00" src="https://user-images.githubusercontent.com/1233880/216574487-b578f337-65e1-467c-be0f-d1c2d659dcfe.png">
</td>
</tr>
<tr>
<td>Showcase: Logged-in<br><br>Plan: Business<br><br>Current theme: Legacy theme</td>
<td>
<img width="1005" alt="Screenshot 2023-02-02 at 14 52 14" src="https://user-images.githubusercontent.com/1233880/216343142-7ce3f302-0b86-4371-be7a-e30db5e51b93.png">
</td>
<td>
<img width="1005" alt="Screenshot 2023-02-03 at 11 28 35" src="https://user-images.githubusercontent.com/1233880/216578168-3d538ba2-b105-4101-803b-fb8ade0bdbc8.png">
</td>
</tr>
<tr>
<td>Showcase: Logged-out</td>
<td>
<img width="1279" alt="Screenshot 2023-02-02 at 14 56 13" src="https://user-images.githubusercontent.com/1233880/216343973-49316e06-3f2b-48ab-96cf-241ae9a464b8.png">
</td>
<td>
<img width="1278" alt="Screenshot 2023-02-02 at 14 56 59" src="https://user-images.githubusercontent.com/1233880/216344166-226838ac-37a1-4bdc-bac7-44cbe9cf1a74.png">
</td>
</tr>
</tbody>
</table>

**Events**

The following events a tracked when a user clicks on any of the options:
- Design your own theme / homepage: 
  - Event: `calypso_themeshowcase_more_options_design_homepage_click`
  - Props:
    - `site_plan`: The site plan.
    - `search_term`: The term searched.
    - `destination`: Where we are sending the user to. The only possible value right now is 'site-editor', but it can be extended in the future once we can start the Pattern Assembler.
- Do it for me:
  - Event: `calypso_themeshowcase_more_options_difm_click`
  - Props:
    - `site_plan`: The site plan.
    - `search_term`: The term searched.
- Upload a theme:
  - Event: `calypso_themeshowcase_more_options_upload_theme_click`
  - Props:
    - `site_plan`: The site plan.
    - `search_term`: The term searched.
    - `destination`: Where we are sending the user to. Possible values are `signup` (for the logged-out showcase), `upload-theme` if the site has an eligible plan, or `checkout` if the site doesn't have an eligible plan.

#### Testing Instructions

- Use the Calypso live link below
- Switch to a Free site with a block theme active
- Go to Appearance > Themes
- Enter a term that doesn't produce any result
- Observe the new card
- Make sure it includes 3 options: Design your own homepage, Do it for me, Upload theme
- Make sure it looks good and the copy is clear
- Make sure all actions redirect to the appropriate place
- Make sure it is responsive on mobile viewports
- Switch to a Simple site with a Business plan and a block theme active
- Make sure the "Upload theme" action no longer upsells the Business plan
- Make sure the "Upload theme" action sends users to `/theme/upload/:site`
- Activate a legacy theme
- Make sure the "Design your own homepage" action is no longer visible
- Make sure the "Upload theme" action sends users to `/wp-admin/theme-install.php`
- Switch to an Atomic site with a Business plan
- Open the `/themes` path on an incognito window
- Enter a term that doesn't produce any result
- Make sure it includes 2 options: Do it for me, Upload theme
- Make sure the "Upload theme" action initiates the Business plan signup flow

mmtr committed:
My Home: Complete "Update your site's design" upon clicking action button

mmtr committed:
Remove no longer needed styles

mmtr committed:
Show card at the bottom of every search

mmtr committed:
Fix tests

BogdanUngureanu commented:
Nice work, Miguel! I've tested it for both logged-in and logged-out users and I noticed some small issues.

**Logged in users, on an Atomic site, the upload theme button is not working.**
Steps to reproduce:
- Go to WordPress.com/themes/your-atomic-site
- Search for "foo"
- Go to the new section and click on the "Upload theme" button
- Notice that nothing happens, you aren't redirected to the upload screen

**Logged out users: The previous upgrade nudge is no longer displayed**
- Open an incognito window and go to /themes
- Search for a random string that doesn't have any results
- Notice that we no longer display any upgrade CTA banner

Is this on purpose? :)

BogdanUngureanu commented:
Two other remarks:
- The previous CTA banner had a tracking event. Maybe we should add tracking events for each of these buttons? Since on mobile we have a different button for the pattern assembler, maybe we should pass a is_mobile parameter (or can we determine it based on the global variables) to differentiate between them?
-  Since on mobile we have a fallback to the Site Editor - which IMHO doesn't bring much value in the context of theme selection since the user is interested in selecting a theme for their site, maybe we should re-order the buttons? Would it make more sense to display the `Upload a theme` first?

mmtr commented on client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx:
```diff
@@ -61,9 +62,18 @@ const PatternAssembler: Step = ( { navigation, flow } ) => {
-		// Require to start the flow from the first step
 		if ( ! selectedDesign ) {
-			goToStep?.( 'goals' );
+			setSelectedDesign( {
+				slug: 'blank-canvas-3',
+				title: 'Blank Canvas',
+				recipe: {
+					stylesheet: 'pub/blank-canvas-3',
+				},
+				verticalizable: false,
+				is_premium: false,
+				is_bundled_with_woo_commerce: false,
+				design_type: 'assembler',
+			} as Design );
```
@Automattic/lego this might conflict with your work done in https://github.com/Automattic/wp-calypso/pull/71852. This was needed to allow users coming from the themes showcase to start the pattern assembler flow while skipping the rest of the site setup flow. Do you see any issues with this approach?

miksansegundo replied:
Thanks for the ping, Miguel!

We added that redirection to the `goals` step to ensure users _creating a new site_ don't skip the goals step if they access the pattern assembler URL directly. Now that there are other use cases like yours to access the assembler step directly on existing sites, we'll have to add an exception to that check and create a new flow with only the assembler.

For now, rather than remove the redirection you could use a URL param to identify the new use case and skip the redirection. In the future, we could use the new assembler flow name to skip the redirection. I'll keep you updated.

bangank36 replied:
> URL param to identify the new use case

I think this would be a better idea for now

mmtr replied:
This has been already addressed in https://github.com/Automattic/wp-calypso/pull/72747, so I'll revert these changes.

mmtr commented on client/landing/stepper/declarative-flow/site-setup-flow.ts:
```diff
@@ -479,6 +479,13 @@ const siteSetupFlow: Flow = {
 					return navigate( 'vertical' );
 
 				case 'patternAssembler':
+					// eslint-disable-next-line no-case-declarations
+					const backTo = urlQueryParams.get( 'backTo' );
+
+					if ( backTo ) {
+						return redirect( backTo );
+					}
+
```
@Automattic/lego same here, this might also conflict with your work done in https://github.com/Automattic/wp-calypso/pull/71852. This was needed to allow users to go back to the themes showcase from the pattern assembler flow if they click the back button. Do you see any issues with this approach?

miksansegundo replied:
I don't see any issues. Thanks for sharing it! We could use it too.

mmtr committed:
Fix upload theme button for Atomic sites

mmtr commented:
> Logged in users, on an Atomic site, the upload theme button is not working.

Nice catch! Fixed in https://github.com/Automattic/wp-calypso/pull/72286/commits/901f84bd73aa04fa5ffb9ce3d3550d0e99c3a6c9

> Logged out users: The previous upgrade nudge is no longer displayed

Yeah, this is in purpose because there isn't a equivalent nudge / action in the new footer, so I opted for removing it for logged out users. 

I'm open to bring it back or to add and alternative nudge / action in the new footer, but it probably needs a discussion with @Copons, @ianstewart or @SaxonF. 

> The previous CTA banner had a tracking event. Maybe we should add tracking events for each of these buttons?

Yup, I also noted that, and ultimately considered that we can make that in a separate PR since we cannot keep using the existing tracking event from the previous banner. But happy to handle it now if folks think otherwise.

> Since on mobile we have a fallback to the Site Editor - which IMHO doesn't bring much value in the context of theme selection since the user is interested in selecting a theme for their site, maybe we should re-order the buttons? Would it make more sense to display the Upload a theme first?

While I agree that the site editor fallback is not great, I'm not sure that the upload theme is a better option for mobile users, since they likely won't have a .zip file at hand.

dsas commented:
How much use does the existing 'no results' footer see on the logged out showcase? 🤔 . My intuition is that this is probably a low priority problem that doesn't have to block this PR and we can file in a separate issue and prioritise against the rest of the backlog.

----

Once we will include dotorg results in the search, zero search results will make the existing footer nonsensical - no matching dotorg themes will exist so upgrading won't help. Definitely the right decision to remove it.

On the new footer actions:
 * We can't link to the pattern assembler without having a site. Maybe we could add a url param that onboarding uses to highlight / pre-select pattern assembler.
 * We could link to DIFM
 * We could mention that you can upload your own theme you get from a 3rd party marketplace / own designer by buying the business plan, linking to https://wordpress.com/start/business/ 

Perhaps for now we could replace the pattern assembler nudge with a reminder that you can find a "close enough" theme and extensively customise it in the site-editor / customizer 🤔 

Maybe it would be interesting to link to a logged out version so people can have a play with it, but that would be a whole different project and Marvel has higher priorities.

BogdanUngureanu committed:
Add PlanGetStartedCTA for logged-out themes page

BogdanUngureanu committed:
Fix the styles for the get started banner.

BogdanUngureanu commented:
Since it drove conversions, I've re-added the Upgrade banner for logged-out themes page so it's RfR again.

@dsas I think we should leave it as it is for now.

dsas commented on client/components/themes-list/index.jsx:
```diff
@@ -130,70 +139,148 @@ function ThemeBlock( props ) {
+				{ translate( "Can't find what you're looking for?" ) }
+			</div>
+			<div className="themes-list__footer-subheading">
+				{ translate( 'Here are a few more options' ) }
```
Non-blocking nitpick. Should this have a colon? 🤔

mmtr replied:
Yup, I wondered the same when I saw the proposed copy in https://github.com/Automattic/dotcom-forge/issues/1453. I think a colon it's more consistent with the rest of our writing style, so I'll add it.

Copons replied:
That's not a colon though! 😛

mmtr replied:
Oh gosh, I was completely lost in translation and genuinely thought for a moment that a colon was the same than a dot 🤦

dsas commented on client/components/themes-list/index.jsx:
```diff
@@ -130,70 +139,148 @@ function ThemeBlock( props ) {
+				<div className="themes-list__footer-action-content">
+					<div className="themes-list__footer-action-text">
+						<div className="themes-list__footer-action-title">
+							{ translate( 'Hire our team of experts to design one for you' ) }
```
All of the other titles explicitly say 'theme': "Create your own theme from scratch", "Upload your theme". Should this say "Hire our team of experts to design a theme for you"?

dsas replied:
Actually maybe this is intentionally vague, lets leave it.

DIFM doesn't design a theme for you. You have to choose a theme from the even more limited list offered, or let us choose one of them for you. DIFM will then make custom page layouts for you.

mmtr replied:
> Actually maybe this is intentionally vague, lets leave it.

You made a good point about being vague though, since translators for example might have trouble understanding what "one" means in this context, so we can probably add a translator comment to clarify.

Copons replied:
+1 for being vague and +1 for adding a translator comment! 👍

mmtr committed:
Add colon to sentence

mmtr committed:
Add comment for translators

mmtr committed:
Use actual colon :)

autumnfjeld commented:
See comment here on path names:  https://github.com/Automattic/wp-calypso/pull/72747#discussion_r1090458726.

So we stay in sync on best path name.  Welcome input. :)

autumnfjeld commented:
To be clear, this flow is a theme switch and we have decided that it is ok to give no warning about what will happen with user's existing site?  (Let me know if I am misunderstanding.)  cc: @Automattic/lego 

Uploading Screen Capture on 2023-01-31 at 12-20-15.mov…


Uploading Screen Capture on 2023-01-31 at 12-20-15.mov…


Uploading Screen Capture on 2023-01-31 at 12-20-15.mov…


Uploading Screen Capture on 2023-01-31 at 12-20-15.mov…

autumnfjeld commented:
One test:
Starting site:
![image](https://user-images.githubusercontent.com/5665959/215635688-53869e4e-6375-4624-ae55-f077b49f941b.png)


What i select in PA
![image](https://user-images.githubusercontent.com/5665959/215635850-82e95ccb-8352-4cf8-85e2-26087c871bb3.png)

![image](https://user-images.githubusercontent.com/5665959/215635880-821964c8-52f0-417a-933d-0767bba6985a.png)


Landing in the Site Editor
![image](https://user-images.githubusercontent.com/5665959/215636031-2f97546a-1ac6-41f6-acdb-37625145e714.png)
![image](https://user-images.githubusercontent.com/5665959/215636031-2f97546a-1ac6-41f6-acdb-37625145e714.png)


This test worked: Classic -> BCPA

A previous test I did had wonky results, what I saw in the site editor didn't match up to what I selected in the PA.  Just an indication of thorough theme switch testing needed.

Copons commented:
As discussed internally (p1675128979469429-slack-C048CUFRGFQ), the PA flow as a theme switch is not ready yet, and we should remove it from these site actions for the time being.

cc @mmtr 

I think it's not a big deal, design-wise. The site actions work regardless. Maybe we should consider if moving the primary to another action, or just keep all of them secondary for now.

mmtr committed:
- Remove pattern assembler from actions
- Start signup flow for logged-out users
- Start site editor for logged-in users
- Show footer for logged-out users

mmtr committed:
Remove unused prop

mmtr commented:
Alright, I added the changes below, so this should be ready for review again:

- The footer is now visible in the logged-out showcase as well, effectively removing the upgrade nudge that @BogdanUngureanu re-added in https://github.com/Automattic/wp-calypso/pull/72286#issuecomment-1405084708.
- The "Create your own theme" action no longer start the pattern assembler flow. Instead, it redirects logged-in users to the site editor (since the pattern assembler is still not ready to be used within the logged-in showcase), while logged-out users are redirected to the `with-theme` signup flow to create a new site with the Blank Canvas theme (which includes the Pattern Assembler step if the `pattern-assembler/logged-out-showcase` feature flag is enabled).
- The "Upload your theme" action redirects logged-in users to either the checkout (with the Business plan automatically added to the cart) or to the install theme page if the site already has the Business plan. Logged-out users are redirected to the signup flow that creates a new Business site.

mmtr committed:
Fix tests

BogdanUngureanu commented:
Works great for me! 🎉 One thing I noticed though (non-blocking): the `Get started` button (the third one) is slightly smaller than the other ones. Should we make it with the the same width?

mmtr commented:
> the `Get started` button (the third one) is slightly smaller than the other ones. Should we make it with the the same width?

Yeah, the width is adapted to its content, as seen in the original design available in https://github.com/Automattic/dotcom-forge/issues/1453, so I think we can leave it as-is.

mmtr committed:
Merge remote-tracking branch 'origin/trunk' into update/empty-themes-search

# Conflicts:
#	client/components/themes-list/index.jsx

mmtr commented:
I just realized the new footer added here conflicts with the new Pattern Assembler CTA introduced in https://github.com/Automattic/wp-calypso/pull/72752, so I asked in https://github.com/Automattic/dotcom-forge/issues/1453#issuecomment-1411951568 if we should drop the CTA in favor of the new footer.

mmtr committed:
Improve wording

mmtr committed:
Remove unnecessary CSS

mmtr committed:
Add tracking events

mmtr committed:
Rename component to MoreOptions

mmtr committed:
Update snapshot

mmtr committed:
Improve responsiveness

autumnfjeld commented:
> drop the CTA in favor of the new footer.

No.  😄    Per discussion p1675291431107489/1675248776.600559-slack-C048CUFRGFQ

Thanks guys.

mmtr committed:
Merge remote-tracking branch 'origin/trunk' into update/empty-themes-search

Copons commented:
> > drop the CTA in favor of the new footer.
> 
> No. 😄 Per discussion p1675291431107489/1675248776.600559-slack-C048CUFRGFQ
> 
> Thanks guys.

The big Lego vs T-Rex fight IS ON!

![https://media.tenor.com/sRqg1X01lsEAAAAC/seinfeld-jason-alexander.gif](https://media.tenor.com/sRqg1X01lsEAAAAC/seinfeld-jason-alexander.gif)

mmtr committed:
Do not show multi-options card when Pattern Assembler CTA shows up

mmtr commented:
Alright, I just pushed an update that hides the new multi-options card when the Pattern Assembler CTA is visible. I also removed any option from the card that was starting the Pattern Assembler.

So it looks like this now:

<table>
<thead><tr><th>Conditions</th><th>Before</th><th>After</th></tr></thead>
<tbody>
<tr>
<td><ul><li>Logged-in showcase</li><li>WP.com themes</li><li>Free plan</li></ul></td>
<td>
<img width="1004" alt="Screenshot 2023-02-02 at 14 51 05" src="https://user-images.githubusercontent.com/1233880/216342893-80ce78af-0ca0-4a95-8966-1b1a9a0a0b54.png">
</td>
<td>
<img width="1009" alt="Screenshot 2023-02-02 at 14 50 35" src="https://user-images.githubusercontent.com/1233880/216342751-81cbfef1-b071-4bc9-916c-9655a5a14a58.png">
</td>
</tr>
<tr>
<td><ul><li>Logged-in showcase</li><li>Empty search</li><li>Free plan</li></ul></td>
<td>
<img width="1001" alt="Screenshot 2023-02-02 at 14 49 25" src="https://user-images.githubusercontent.com/1233880/216342469-a324fc56-c6b2-4389-b608-a2987d88fa0f.png">
</td>
<td>
<img width="1006" alt="Screenshot 2023-02-02 at 14 49 02" src="https://user-images.githubusercontent.com/1233880/216342357-2490892b-2b3b-4f85-89a5-d200d24ac124.png">
</td>
</tr>
<tr>
<td><ul><li>Logged-in showcase</li><li>Empty search</li><li>Business plan</li></ul></td>
<td>
<img width="1005" alt="Screenshot 2023-02-02 at 14 52 14" src="https://user-images.githubusercontent.com/1233880/216343142-7ce3f302-0b86-4371-be7a-e30db5e51b93.png">
</td>
<td>
<img width="1006" alt="Screenshot 2023-02-02 at 14 52 30" src="https://user-images.githubusercontent.com/1233880/216343188-8340a540-1431-4890-a77e-7b9235943555.png">
</td>
</tr>
</tr>
<tr>
<td><ul><li>Logged-in showcase</li><li>WP.org theme</li><li>Free plan</li></ul></td>
<td>
<img width="1004" alt="Screenshot 2023-02-02 at 14 54 12" src="https://user-images.githubusercontent.com/1233880/216343525-a10539ef-f98d-4e69-a497-0945240a934c.png">
</td>
<td>
<img width="1006" alt="Screenshot 2023-02-02 at 14 54 50" src="https://user-images.githubusercontent.com/1233880/216343650-9e1dd06e-d1d8-4f19-a6b5-fc99504c3764.png">
</td>
</tr>
<tr>
<td><ul><li>Logged-out showcase</li><li>Empty</li></ul></td>
<td>
<img width="1279" alt="Screenshot 2023-02-02 at 14 56 13" src="https://user-images.githubusercontent.com/1233880/216343973-49316e06-3f2b-48ab-96cf-241ae9a464b8.png">
</td>
<td>
<img width="1278" alt="Screenshot 2023-02-02 at 14 56 59" src="https://user-images.githubusercontent.com/1233880/216344166-226838ac-37a1-4bdc-bac7-44cbe9cf1a74.png">
</td>
</tr>
<tr>
<td><ul><li>Logged-out showcase</li><li>WP.com themes</li><li><code>pattern-assembler/logged-out-showcase</code> feature flag disabled</li></ul></td>
<td>
<img width="1280" alt="Screenshot 2023-02-02 at 14 57 36" src="https://user-images.githubusercontent.com/1233880/216344724-87632224-e2b8-4d97-a2b2-f156b3fe09d0.png">
</td>
<td>
<img width="1272" alt="Screenshot 2023-02-02 at 14 58 11" src="https://user-images.githubusercontent.com/1233880/216344753-c4b8d7c8-ec88-4068-8669-703e03613568.png">
</td>
</tr>
<tr>
<td><ul><li>Logged-out showcase</li><li>WP.com themes</li><li><code>pattern-assembler/logged-out-showcase</code> feature flag enabled</li></ul></td>
<td>
<img width="1275" alt="Screenshot 2023-02-02 at 14 59 57" src="https://user-images.githubusercontent.com/1233880/216344956-0da685ea-2bc3-4e5f-a148-ea009445abd3.png">
</td>
<td align=center>
No changes
<img width="1271" alt="Screenshot 2023-02-02 at 15 00 14" src="https://user-images.githubusercontent.com/1233880/216344985-d5ddd80b-205c-418f-abab-ce2142d841ce.png">
</td>
</tr>
</tbody>
</table>

mmtr committed:
Only show the new multi-options card when there are no results

mmtr committed:
Prevent double-upsell situation

mmtr committed:
Merge remote-tracking branch 'origin/trunk' into update/empty-themes-search

# Conflicts:
#	client/components/themes-list/index.jsx

mmtr commented:
As per feedback in p1675348333564989/1675248776.600559-slack-C048CUFRGFQ, I pushed another update to exclude the new multi-options card from searches with results, since the removal of the pattern assembler entry point doesn't make it worthwhile.

I also added a small fix to prevent upsell banners from appearing if the card shows up.

mmtr committed:
Exclude "Design your own homepage" option if the site editor is unavailable

mmtr committed:
Update unit test snapshot

mmtr committed:
Remove no longer needed mocks

Copons commented on client/components/themes-list/index.jsx:
```diff
@@ -148,70 +159,165 @@ function ThemeBlock( props ) {
 
-function Empty( props ) {
-	const { wpOrgThemes, emptyContent, searchTerm, upsellCardDisplayed, translate } = props;
+function Options( { isFSEActive, recordTracksEvent, searchTerm, translate, upsellCardDisplayed } ) {
```
This is a pretty big component; what do you think of moving it to its own file?

Copons replied:
We might even simplify the props by importing them into the end-consumer component rather than dragging them across multiple depth levels, but let's not overcomplicate this PR any further.

Copons commented on client/components/themes-list/index.jsx:
```diff
@@ -148,70 +159,165 @@ function ThemeBlock( props ) {
+	options.push( {
+		title: translate( 'Hire our team of experts to design one for you', {
+			comment:
+				'"One" means a theme in this context (i.e. "Hire our team of experts to design a theme for you")',
```
IIRC, we chose to keep "one" vague as it can mean both "theme" and "site".
Probably not a big deal for translators, but we might want to mention this in the comment.
E.g.
```suggestion
				'"One" could mean "theme" or "site" in this context (i.e. "Hire our team of experts to design a theme for you")',
```

dsas commented:
I'm still getting a 'Design your own' when I have empty search results. I'm on a simple site with a business plan
![image](https://user-images.githubusercontent.com/93301/216595920-541c7e92-bc22-4653-a931-8165d5047a94.png)

Copons reviewed (approved):
This LGTM.

The only feedback I have is fundamentally about past code decisions; we can ignore them in this PR.

I also have one request: let's record all the new events somewhere!
Even just adding them to the PR description is fine (I can take care of reporting them to the PT).

And, of course, don't forget to register them once they have been used! 🙂

mmtr commented:
> I also have one request: let's record all the new events somewhere!
> Even just adding them to the PR description is fine (I can take care of reporting them to the PT).

Done!

mmtr committed:
Update client/components/themes-list/index.jsx

Co-authored-by: Jacopo Tomasone <Copons@users.noreply.github.com>

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7793677

Thank you @mmtr for including a screenshot in the description! This is really helpful for our translators.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7793677).
